internal/api/grpc/server/middleware/execution_interceptor.go

```diff
@@ -146,9 +142,9 @@ func (c *ContextInfoResponse) GetHTTPRequestBody() []byte {
 }
 
 func (c *ContextInfoResponse) SetHTTPResponseBody(resp []byte) error {
-	return json.Unmarshal(resp, c.originalResponse)
+	return json.Unmarshal(resp, c.Response.Message)
```

Comment:
as suggested last time:
```suggestion
	return json.Unmarshal(resp, &c.Response)
```